tony-history-server/app/utils/HdfsUtils.java

```diff
@@ -0,0 +1,108 @@
+package utils;
+
+import java.io.BufferedReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import org.apache.commons.io.IOUtils;
+import org.apache.hadoop.fs.FSDataInputStream;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import play.Logger;
+
+
+/**
+ * The class handles all HDFS file operations
+ */
+public class HdfsUtils {
```

Comment:
this seems generic for all modules. We might need a component for all utils. Something like hadoop-common.